main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -69,6 +69,27 @@ DataTableColumnHeaderUI.prototype._render = function() {
   elmts.dropdownMenu.on('click',function() {
     self._createMenuForColumnHeader(this);
   });
+  var serviceUrl = null;
+var service = null;
+var serviceLogo=null;
+if (this._column.reconConfig) {
+serviceUrl =this._column.reconConfig.service;
+}
+if (serviceUrl) {
+service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+}
+if(service){
+serviceLogo=service.logo;
+}
+
+var img =$("<img>");
+if(serviceLogo ){
+var imageUrl = serviceLogo;
+img.attr("src", imageUrl);
+img.attr("title", service.name);
+img.addClass("serviceLogo")
+img.appendTo(elmts.serviceLogoContainer.show());
+}
```

Comment:
Those changes should be indented properly (looking at the code around, each indentation level is two space characters).